### ISSUE TITLE
Can't delete messages from inbox

### ISSUE THREAD

**Issue by cgarwood82** [nextcloud/mail#882]
### Steps to reproduce
1. Delete mail message and see it removed from list of messages
2. Click to file viewer in Nextcloud
3. Come back to mail
4. Message is still there

### Expected behaviour
When I click the trash can, the message should go to the trash.

### Actual behaviour
The message appears to delete. But if I click away to somewhere else on nextcloud and return to mail, the messages are still there and marked Unread. 

### Mail app

**Mail app version:**  0.7.10

**Mailserver or service:** GoDaddy Professional Email

**Number of accounts:** 1

### Server configuration

**Operating system**:  
3.10.0-693.21.1.el7.x86_64
CentOS Linux release 7.4.1708 (Core) 

**Web server:**
httpd-2.4.6-67.el7.centos.6.x86_64

**Database:**
mariadb-5.5.56-2.el7.x86_64

**PHP version:**
php-7.2.5-1.el7.remi.x86_64

**Version:** (see admin page)
Nextcloud 13.0.2

**Updated from an older version or fresh install:**
Fresh Install

**List of activated apps:**

<details>

Enabled:
  - activity: 2.6.1
  - bruteforcesettings: 1.0.3
  - calendar: 1.6.1
  - comments: 1.3.0
  - contacts: 2.1.3
  - dav: 1.4.6
  - external: 3.0.2
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_pdfviewer: 1.2.1
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - gallery: 18.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - mail: 0.7.10
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.2
  - oauth2: 1.1.0
  - password_policy: 1.3.0
  - provisioning_api: 1.3.0
  - richdocuments: 2.0.8
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - socialsharing_diaspora: 1.0.2
  - socialsharing_email: 1.0.3
  - socialsharing_facebook: 1.0.2
  - socialsharing_googleplus: 1.0.2
  - socialsharing_twitter: 1.0.2
  - spreed: 3.1.0
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - theming: 1.4.1
  - twofactor_backupcodes: 1.2.3
  - unsplash: 1.1.1
  - updatenotification: 1.3.0
  - workflowengine: 1.3.0

</details>

**The content of config/config.php:**

<details>

```

{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "board.REMOVED.org"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/board.REMOVED.org",
        "dbtype": "mysql",
        "version": "13.0.2.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "sendmail",
        "mail_smtpauthtype": "LOGIN",
        "mail_domain": "***REMOVED SENSITIVE VALUE***"
    }
}

```

</details>

#### Client configuration
**Browser:** Firefox 59.0.2 (64-bit)

**Operating system:** Arch Linux

##### Web server error log
<details>

```

[Fri Apr 27 15:25:57.419030 2018] [authz_core:error] [pid 22293] [client 108.666.666.666:53037] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 16:22:36.726653 2018] [authz_core:error] [pid 24081] [client 108.666.666.666:57251] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 16:22:41.303945 2018] [authz_core:error] [pid 24072] [client 108.666.666.666:50683] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 22:19:53.076675 2018] [authz_core:error] [pid 25493] [client 108.666.666.666:26999] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 22:20:19.455067 2018] [authz_core:error] [pid 25492] [client 108.666.666.666:61320] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:02:55.509243 2018] [authz_core:error] [pid 25497] [client 108.666.666.666:62884] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:04:22.242504 2018] [authz_core:error] [pid 27246] [client 108.666.666.666:27686] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:13:43.348009 2018] [authz_core:error] [pid 24723] [client 108.666.666.666:47723] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:14:04.323528 2018] [authz_core:error] [pid 27727] [client 108.666.666.666:19274] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:21:15.770080 2018] [authz_core:error] [pid 27774] [client 108.666.666.666:41936] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:24:04.478237 2018] [authz_core:error] [pid 24723] [client 108.666.666.666:28382] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:24:16.036729 2018] [authz_core:error] [pid 27791] [client 108.666.666.666:13621] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:24:21.502816 2018] [authz_core:error] [pid 28012] [client 108.666.666.666:55356] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:25:04.862692 2018] [authz_core:error] [pid 27281] [client 108.666.666.666:20422] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:25:08.065215 2018] [authz_core:error] [pid 27986] [client 108.666.666.666:22944] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:25:08.929987 2018] [authz_core:error] [pid 28054] [client 108.666.666.666:48520] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:36:24.841870 2018] [authz_core:error] [pid 28120] [client 108.666.666.666:59597] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:41:07.694875 2018] [authz_core:error] [pid 28093] [client 108.666.666.666:57195] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Fri Apr 27 23:41:10.350156 2018] [authz_core:error] [pid 28183] [client 108.666.666.666:12471] AH01630: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata

```

</details>

##### Server log (data/nextcloud.log)
<details>

```
Error: Call to a member function getEmail() on null

    /var/www/html/nextcloud/apps/mail/lib/Mailbox.php - line 205: OCA\Mail\Model\IMAPMessage->jsonSerialize()
    /var/www/html/nextcloud/apps/mail/lib/Controller/MessagesController.php - line 129: OCA\Mail\Mailbox->getMessages('', NULL)
    [internal function] OCA\Mail\Controller\MessagesController->index(2, 'U2VudCBJdGVtcw=...', NULL, '')
    /var/www/html/nextcloud/lib/private/AppFramework/Http/Dispatcher.php - line 161: call_user_func_array(Array, Array)
    /var/www/html/nextcloud/lib/private/AppFramework/Http/Dispatcher.php - line 91: OC\AppFramework\Http\Dispatcher->executeController(Object(OCA\Mail\Controller\MessagesController), 'index')
    /var/www/html/nextcloud/lib/private/AppFramework/App.php - line 115: OC\AppFramework\Http\Dispatcher->dispatch(Object(OCA\Mail\Controller\MessagesController), 'index')
    /var/www/html/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php - line 47: OC\AppFramework\App main('OCA\\Mail\\Contro...', 'index', Object(OC\AppFramework\DependencyInjection\DIContainer), Array)
    [internal function] OC\AppFramework\Routing\RouteActionHandler->__invoke(Array)
    /var/www/html/nextcloud/lib/private/Route/Router.php - line 297: call_user_func(Object(OC\AppFramework\Routing\RouteActionHandler), Array)
    /var/www/html/nextcloud/lib/base.php - line 999: OC\Route\Router->match('/apps/mail/api/...')
    /var/www/html/nextcloud/index.php - line 37: OC handleRequest()
    {main}

```
</details>

##### IMAP log (data/horde_imap.log)

I deleted the email, clicked out of the inbox, clicked out of mail, clicked back into mail, and I think I deleted the mail again when it showed up.

<details>

```
------------------------------
>> Sat, 28 Apr 2018 04:47:09 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2314 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: REMOVED
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2571 seconds.
C: 2 EXAMINE INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.0804 seconds.
C: 3 UID FETCH 1:* (INTERNALDATE)
S: * 1 FETCH (UID 1 INTERNALDATE "28-Apr-2018 04:44:26 +0000")
S: 3 OK FETCH completed.
>> Command 3 took 0.0878 seconds.
C: 4 UID FETCH 1 (ENVELOPE FLAGS RFC822.SIZE INTERNALDATE BODYSTRUCTURE BODY.PEEK[HEADER.FIELDS (IMPORTANCE LIST-POST X-PRIORITY CONTENT-TYPE)])
S: * 1 FETCH (UID 1 ENVELOPE ("Sat, 28 Apr 2018 00:44:22 -0400" "test" (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("comms@REMOVED.org" NIL "comms" "REMOVED.org")) NIL NIL NIL "<BulbjKT9UTzpABV2JPdpncSrHJUfwzy2JgbTvvyhQGlOoxb9Hv2A2JLidcUmHkTLDryyrpJnjB5qYdVpLjWkjAHOYubbjqbxCyaX0UYwM2k=@removed.com>") FLAGS () RFC822.SIZE 2602 INTERNALDATE "28-Apr-2018 04:44:26 +0000" BODYSTRUCTURE (("text" "plain" ("charset" "utf-8") NIL NIL "base64" 12 2 NIL NIL NIL)("text" "html" ("charset" "utf-8") NIL NIL "base64" 382 7 NIL NIL NIL) "alternative" ("boundary" "b1_1db589e142f867cb3fcde8d6834fb8cd") NIL NIL) BODY[HEADER.FIELDS ("IMPORTANCE" "LIST-POST" "X-PRIORITY" "CONTENT-TYPE")] {89}
S: [LITERAL DATA: 89 bytes]
S: )
S: 4 OK FETCH completed.
>> Command 4 took 0.2071 seconds.
C: 5 LOGOUT
S: * BYE IMAP server shutting down
S: 5 OK LOGOUT completed
>> Command 5 took 0.0763 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:47:10 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2332 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.25 seconds.
C: 2 EXAMINE INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.0798 seconds.
C: 3 UID FETCH 1 (ENVELOPE BODYSTRUCTURE FLAGS RFC822.SIZE INTERNALDATE BODY.PEEK[HEADER.FIELDS (IMPORTANCE LIST-POST X-PRIORITY CONTENT-TYPE)])
S: * 1 FETCH (UID 1 ENVELOPE ("Sat, 28 Apr 2018 00:44:22 -0400" "test" (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("comms@REMOVED.org" NIL "comms" "REMOVED.org")) NIL NIL NIL "<BulbjKT9UTzpABV2JPdpncSrHJUfwzy2JgbTvvyhQGlOoxb9Hv2A2JLidcUmHkTLDryyrpJnjB5qYdVpLjWkjAHOYubbjqbxCyaX0UYwM2k=@removed.com>") BODYSTRUCTURE (("text" "plain" ("charset" "utf-8") NIL NIL "base64" 12 2 NIL NIL NIL)("text" "html" ("charset" "utf-8") NIL NIL "base64" 382 7 NIL NIL NIL) "alternative" ("boundary" "b1_1db589e142f867cb3fcde8d6834fb8cd") NIL NIL) FLAGS () RFC822.SIZE 2602 INTERNALDATE "28-Apr-2018 04:44:26 +0000" BODY[HEADER.FIELDS ("IMPORTANCE" "LIST-POST" "X-PRIORITY" "CONTENT-TYPE")] {89}
S: [LITERAL DATA: 89 bytes]
S: )
S: 3 OK FETCH completed.
>> Command 3 took 0.2048 seconds.
C: 4 UID FETCH 1 (BODY.PEEK[1] BODY.PEEK[1.MIME])
S: * 1 FETCH (UID 1 BODY[1] {12}
S: [LITERAL DATA: 12 bytes]
S:  BODY[1.MIME] {78}
S: [LITERAL DATA: 78 bytes]
S: )
S: 4 OK FETCH completed.
>> Command 4 took 0.1921 seconds.
C: 5 LOGOUT
S: * BYE IMAP server shutting down
S: 5 OK LOGOUT completed
>> Command 5 took 0.0763 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:47:11 +0000
>> Connection to: imap://imap.REMOVED.net:993/
------------------------------
>> Sat, 28 Apr 2018 04:47:11 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2337 seconds.
>> Server connection took 0.2304 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2612 seconds.
C: 2 EXAMINE INBOX
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2549 seconds.
C: 2 SELECT INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.0803 seconds.
C: 3 UID FETCH 1 (ENVELOPE BODYSTRUCTURE FLAGS RFC822.SIZE INTERNALDATE BODY.PEEK[HEADER.FIELDS (IMPORTANCE LIST-POST X-PRIORITY CONTENT-TYPE)])
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-WRITE] SELECT completed.
>> Command 2 took 0.0798 seconds.
C: 3 UID STORE 1 +FLAGS \seen
S: 3 NO Error in IMAP command received by server.
>> Command 3 took 0.0761 seconds.
C: 4 LOGOUT
S: * BYE IMAP server shutting down
S: 4 OK LOGOUT completed
>> Command 4 took 0.0766 seconds.
S: * 1 FETCH (UID 1 ENVELOPE ("Sat, 28 Apr 2018 00:44:22 -0400" "test" (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("comms@REMOVED.org" NIL "comms" "REMOVED.org")) NIL NIL NIL "<BulbjKT9UTzpABV2JPdpncSrHJUfwzy2JgbTvvyhQGlOoxb9Hv2A2JLidcUmHkTLDryyrpJnjB5qYdVpLjWkjAHOYubbjqbxCyaX0UYwM2k=@removed.com>") BODYSTRUCTURE (("text" "plain" ("charset" "utf-8") NIL NIL "base64" 12 2 NIL NIL NIL)("text" "html" ("charset" "utf-8") NIL NIL "base64" 382 7 NIL NIL NIL) "alternative" ("boundary" "b1_1db589e142f867cb3fcde8d6834fb8cd") NIL NIL) FLAGS () RFC822.SIZE 2602 INTERNALDATE "28-Apr-2018 04:44:26 +0000" BODY[HEADER.FIELDS ("IMPORTANCE" "LIST-POST" "X-PRIORITY" "CONTENT-TYPE")] {89}
S: [LITERAL DATA: 89 bytes]
S: )
S: 3 OK FETCH completed.
>> Command 3 took 0.2081 seconds.
C: 4 UID FETCH 1 (BODY.PEEK[1] BODY.PEEK[1.MIME])
S: * 1 FETCH (UID 1 BODY[1] {12}
S: [LITERAL DATA: 12 bytes]
S:  BODY[1.MIME] {78}
S: [LITERAL DATA: 78 bytes]
S: )
S: 4 OK FETCH completed.
>> Command 4 took 0.1909 seconds.
C: 5 UID FETCH 1 (BODY.PEEK[2] BODY.PEEK[2.MIME])
S: * 1 FETCH (UID 1 BODY[2] {382}
S: [LITERAL DATA: 382 bytes]
S:  BODY[2.MIME] {77}
S: [LITERAL DATA: 77 bytes]
S: )
S: 5 OK FETCH completed.
>> Command 5 took 0.1912 seconds.
C: 6 LOGOUT
S: * BYE IMAP server shutting down
S: 6 OK LOGOUT completed
>> Command 6 took 0.0761 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:47:17 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2319 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2557 seconds.
C: 2 LIST "" *
S: * LIST (\NoInferiors) "/" "INBOX"
S: * LIST (\NoInferiors \Junk) "/" "Bulk Mail"
S: * LIST (\NoInferiors) "/" "Drafts"
S: * LIST (\NoInferiors) "/" "Send_Later"
S: * LIST (\NoInferiors \Sent) "/" "Sent Items"
S: * LIST (\HasNoChildren \Trash) "/" "Trash"
S: 2 OK  Success
>> Command 2 took 0.0818 seconds.
C: 3 SELECT INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 3 OK [READ-WRITE] SELECT completed.
>> Command 3 took 0.0768 seconds.
C: 4 UID COPY 1 Trash
S: 4 OK [COPYUID 1 1 3] Done
>> Command 4 took 0.1084 seconds.
C: 5 UID SEARCH NOT UID 1 DELETED
S: * SEARCH
S: 5 OK SEARCH done.
>> Command 5 took 0.0964 seconds.
C: 6 UID STORE 1 +FLAGS \deleted
S: 6 NO Error in IMAP command received by server.
>> Command 6 took 0.0767 seconds.
C: 7 CLOSE
S: 7 OK mailbox closed.
>> Command 7 took 0.0763 seconds.
C: 8 SELECT INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 8 OK [READ-WRITE] SELECT completed.
>> Command 8 took 0.0815 seconds.
C: 9 LOGOUT
S: * BYE IMAP server shutting down
S: 9 OK LOGOUT completed
>> Command 9 took 0.0764 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:47:34 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2323 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2491 seconds.
C: 2 EXAMINE INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.0799 seconds.
C: 3 UID SEARCH FLAGGED UNDELETED
S: * SEARCH
S: 3 OK SEARCH done.
>> Command 3 took 0.0851 seconds.
C: 4 LOGOUT
S: * BYE IMAP server shutting down
S: 4 OK LOGOUT completed
>> Command 4 took 0.0764 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:47:44 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2322 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2532 seconds.
C: 2 EXAMINE "Sent Items"
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 0 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 1] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.08 seconds.
C: 3 UID FETCH 1:* (INTERNALDATE)
S: 3 OK FETCH completed.
>> Command 3 took 0.0897 seconds.
C: 4 LOGOUT
S: * BYE IMAP server shutting down
S: 4 OK LOGOUT completed
>> Command 4 took 0.0763 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:47:54 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2303 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2551 seconds.
C: 2 EXAMINE INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.0803 seconds.
C: 3 UID FETCH 1:* (INTERNALDATE)
S: * 1 FETCH (UID 1 INTERNALDATE "28-Apr-2018 04:44:26 +0000")
S: 3 OK FETCH completed.
>> Command 3 took 0.0822 seconds.
C: 4 UID FETCH 1 (ENVELOPE FLAGS RFC822.SIZE INTERNALDATE BODYSTRUCTURE BODY.PEEK[HEADER.FIELDS (IMPORTANCE LIST-POST X-PRIORITY CONTENT-TYPE)])
S: * 1 FETCH (UID 1 ENVELOPE ("Sat, 28 Apr 2018 00:44:22 -0400" "test" (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("comms@REMOVED.org" NIL "comms" "REMOVED.org")) NIL NIL NIL "<BulbjKT9UTzpABV2JPdpncSrHJUfwzy2JgbTvvyhQGlOoxb9Hv2A2JLidcUmHkTLDryyrpJnjB5qYdVpLjWkjAHOYubbjqbxCyaX0UYwM2k=@removed.com>") FLAGS () RFC822.SIZE 2602 INTERNALDATE "28-Apr-2018 04:44:26 +0000" BODYSTRUCTURE (("text" "plain" ("charset" "utf-8") NIL NIL "base64" 12 2 NIL NIL NIL)("text" "html" ("charset" "utf-8") NIL NIL "base64" 382 7 NIL NIL NIL) "alternative" ("boundary" "b1_1db589e142f867cb3fcde8d6834fb8cd") NIL NIL) BODY[HEADER.FIELDS ("IMPORTANCE" "LIST-POST" "X-PRIORITY" "CONTENT-TYPE")] {89}
S: [LITERAL DATA: 89 bytes]
S: )
S: 4 OK FETCH completed.
>> Command 4 took 0.2032 seconds.
C: 5 LOGOUT
S: * BYE IMAP server shutting down
S: 5 OK LOGOUT completed
>> Command 5 took 0.0759 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:47:56 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.231 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2548 seconds.
C: 2 EXAMINE INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.0814 seconds.
C: 3 UID FETCH 1 (ENVELOPE BODYSTRUCTURE FLAGS RFC822.SIZE INTERNALDATE BODY.PEEK[HEADER.FIELDS (IMPORTANCE LIST-POST X-PRIORITY CONTENT-TYPE)])
S: * 1 FETCH (UID 1 ENVELOPE ("Sat, 28 Apr 2018 00:44:22 -0400" "test" (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("comms@REMOVED.org" NIL "comms" "REMOVED.org")) NIL NIL NIL "<BulbjKT9UTzpABV2JPdpncSrHJUfwzy2JgbTvvyhQGlOoxb9Hv2A2JLidcUmHkTLDryyrpJnjB5qYdVpLjWkjAHOYubbjqbxCyaX0UYwM2k=@removed.com>") BODYSTRUCTURE (("text" "plain" ("charset" "utf-8") NIL NIL "base64" 12 2 NIL NIL NIL)("text" "html" ("charset" "utf-8") NIL NIL "base64" 382 7 NIL NIL NIL) "alternative" ("boundary" "b1_1db589e142f867cb3fcde8d6834fb8cd") NIL NIL) FLAGS () RFC822.SIZE 2602 INTERNALDATE "28-Apr-2018 04:44:26 +0000" BODY[HEADER.FIELDS ("IMPORTANCE" "LIST-POST" "X-PRIORITY" "CONTENT-TYPE")] {89}
S: [LITERAL DATA: 89 bytes]
S: )
S: 3 OK FETCH completed.
>> Command 3 took 0.2061 seconds.
C: 4 UID FETCH 1 (BODY.PEEK[1] BODY.PEEK[1.MIME])
S: * 1 FETCH (UID 1 BODY[1] {12}
S: [LITERAL DATA: 12 bytes]
S:  BODY[1.MIME] {78}
S: [LITERAL DATA: 78 bytes]
S: )
S: 4 OK FETCH completed.
>> Command 4 took 0.1917 seconds.
C: 5 LOGOUT
S: * BYE IMAP server shutting down
S: 5 OK LOGOUT completed
>> Command 5 took 0.0759 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:47:57 +0000
>> Connection to: imap://imap.REMOVED.net:993/
------------------------------
>> Sat, 28 Apr 2018 04:47:57 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2339 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
>> Server connection took 0.2332 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2486 seconds.
C: 2 SELECT INBOX
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-WRITE] SELECT completed.
>> Command 2 took 0.0796 seconds.
C: 3 UID STORE 1 +FLAGS \seen
S: 3 NO Error in IMAP command received by server.
>> Command 3 took 0.0763 seconds.
C: 4 LOGOUT
S: * BYE IMAP server shutting down
S: 4 OK LOGOUT completed
>> Command 4 took 0.0766 seconds.
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2511 seconds.
C: 2 EXAMINE INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.0799 seconds.
C: 3 UID FETCH 1 (ENVELOPE BODYSTRUCTURE FLAGS RFC822.SIZE INTERNALDATE BODY.PEEK[HEADER.FIELDS (IMPORTANCE LIST-POST X-PRIORITY CONTENT-TYPE)])
S: * 1 FETCH (UID 1 ENVELOPE ("Sat, 28 Apr 2018 00:44:22 -0400" "test" (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("comms@REMOVED.org" NIL "comms" "REMOVED.org")) NIL NIL NIL "<BulbjKT9UTzpABV2JPdpncSrHJUfwzy2JgbTvvyhQGlOoxb9Hv2A2JLidcUmHkTLDryyrpJnjB5qYdVpLjWkjAHOYubbjqbxCyaX0UYwM2k=@removed.com>") BODYSTRUCTURE (("text" "plain" ("charset" "utf-8") NIL NIL "base64" 12 2 NIL NIL NIL)("text" "html" ("charset" "utf-8") NIL NIL "base64" 382 7 NIL NIL NIL) "alternative" ("boundary" "b1_1db589e142f867cb3fcde8d6834fb8cd") NIL NIL) FLAGS () RFC822.SIZE 2602 INTERNALDATE "28-Apr-2018 04:44:26 +0000" BODY[HEADER.FIELDS ("IMPORTANCE" "LIST-POST" "X-PRIORITY" "CONTENT-TYPE")] {89}
S: [LITERAL DATA: 89 bytes]
S: )
S: 3 OK FETCH completed.
>> Command 3 took 0.2032 seconds.
C: 4 UID FETCH 1 (BODY.PEEK[1] BODY.PEEK[1.MIME])
S: * 1 FETCH (UID 1 BODY[1] {12}
S: [LITERAL DATA: 12 bytes]
S:  BODY[1.MIME] {78}
S: [LITERAL DATA: 78 bytes]
S: )
S: 4 OK FETCH completed.
>> Command 4 took 0.1912 seconds.
C: 5 LOGOUT
S: * BYE IMAP server shutting down
S: 5 OK LOGOUT completed
>> Command 5 took 0.0761 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:48:01 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2297 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2553 seconds.
C: 2 LIST "" *
S: * LIST (\NoInferiors) "/" "INBOX"
S: * LIST (\NoInferiors \Junk) "/" "Bulk Mail"
S: * LIST (\NoInferiors) "/" "Drafts"
S: * LIST (\NoInferiors) "/" "Send_Later"
S: * LIST (\NoInferiors \Sent) "/" "Sent Items"
S: * LIST (\HasNoChildren \Trash) "/" "Trash"
S: 2 OK  Success
>> Command 2 took 0.0799 seconds.
C: 3 SELECT INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 3 OK [READ-WRITE] SELECT completed.
>> Command 3 took 0.0764 seconds.
C: 4 UID COPY 1 Trash
S: 4 OK [COPYUID 1 1 4] Done
>> Command 4 took 0.1054 seconds.
C: 5 UID SEARCH NOT UID 1 DELETED
S: * SEARCH
S: 5 OK SEARCH done.
>> Command 5 took 0.0875 seconds.
C: 6 UID STORE 1 +FLAGS \deleted
S: 6 NO Error in IMAP command received by server.
>> Command 6 took 0.0762 seconds.
C: 7 CLOSE
S: 7 OK mailbox closed.
>> Command 7 took 0.0759 seconds.
C: 8 SELECT INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 8 OK [READ-WRITE] SELECT completed.
>> Command 8 took 0.0828 seconds.
C: 9 LOGOUT
S: * BYE IMAP server shutting down
S: 9 OK LOGOUT completed
>> Command 9 took 0.076 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:48:11 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2308 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.249 seconds.
C: 2 EXAMINE INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.0788 seconds.
C: 3 UID FETCH 1:* (INTERNALDATE)
S: * 1 FETCH (UID 1 INTERNALDATE "28-Apr-2018 04:44:26 +0000")
S: 3 OK FETCH completed.
>> Command 3 took 0.0818 seconds.
C: 4 UID FETCH 1 (ENVELOPE FLAGS RFC822.SIZE INTERNALDATE BODYSTRUCTURE BODY.PEEK[HEADER.FIELDS (IMPORTANCE LIST-POST X-PRIORITY CONTENT-TYPE)])
S: * 1 FETCH (UID 1 ENVELOPE ("Sat, 28 Apr 2018 00:44:22 -0400" "test" (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("comms@REMOVED.org" NIL "comms" "REMOVED.org")) NIL NIL NIL "<BulbjKT9UTzpABV2JPdpncSrHJUfwzy2JgbTvvyhQGlOoxb9Hv2A2JLidcUmHkTLDryyrpJnjB5qYdVpLjWkjAHOYubbjqbxCyaX0UYwM2k=@removed.com>") FLAGS () RFC822.SIZE 2602 INTERNALDATE "28-Apr-2018 04:44:26 +0000" BODYSTRUCTURE (("text" "plain" ("charset" "utf-8") NIL NIL "base64" 12 2 NIL NIL NIL)("text" "html" ("charset" "utf-8") NIL NIL "base64" 382 7 NIL NIL NIL) "alternative" ("boundary" "b1_1db589e142f867cb3fcde8d6834fb8cd") NIL NIL) BODY[HEADER.FIELDS ("IMPORTANCE" "LIST-POST" "X-PRIORITY" "CONTENT-TYPE")] {89}
S: [LITERAL DATA: 89 bytes]
S: )
S: 4 OK FETCH completed.
>> Command 4 took 0.2037 seconds.
C: 5 LOGOUT
S: * BYE IMAP server shutting down
S: 5 OK LOGOUT completed
>> Command 5 took 0.0761 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:48:13 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2297 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.256 seconds.
C: 2 EXAMINE INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-ONLY] EXAMINE completed.
>> Command 2 took 0.08 seconds.
C: 3 UID FETCH 1 (ENVELOPE BODYSTRUCTURE FLAGS RFC822.SIZE INTERNALDATE BODY.PEEK[HEADER.FIELDS (IMPORTANCE LIST-POST X-PRIORITY CONTENT-TYPE)])
S: * 1 FETCH (UID 1 ENVELOPE ("Sat, 28 Apr 2018 00:44:22 -0400" "test" (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("REMOVED" NIL "REMOVED" "removed.com")) (("comms@REMOVED.org" NIL "comms" "REMOVED.org")) NIL NIL NIL "<BulbjKT9UTzpABV2JPdpncSrHJUfwzy2JgbTvvyhQGlOoxb9Hv2A2JLidcUmHkTLDryyrpJnjB5qYdVpLjWkjAHOYubbjqbxCyaX0UYwM2k=@removed.com>") BODYSTRUCTURE (("text" "plain" ("charset" "utf-8") NIL NIL "base64" 12 2 NIL NIL NIL)("text" "html" ("charset" "utf-8") NIL NIL "base64" 382 7 NIL NIL NIL) "alternative" ("boundary" "b1_1db589e142f867cb3fcde8d6834fb8cd") NIL NIL) FLAGS () RFC822.SIZE 2602 INTERNALDATE "28-Apr-2018 04:44:26 +0000" BODY[HEADER.FIELDS ("IMPORTANCE" "LIST-POST" "X-PRIORITY" "CONTENT-TYPE")] {89}
S: [LITERAL DATA: 89 bytes]
S: )
S: 3 OK FETCH completed.
>> Command 3 took 0.2056 seconds.
C: 4 UID FETCH 1 (BODY.PEEK[1] BODY.PEEK[1.MIME])
S: * 1 FETCH (UID 1 BODY[1] {12}
S: [LITERAL DATA: 12 bytes]
S:  BODY[1.MIME] {78}
S: [LITERAL DATA: 78 bytes]
S: )
S: 4 OK FETCH completed.
>> Command 4 took 0.1919 seconds.
C: 5 LOGOUT
S: * BYE IMAP server shutting down
S: 5 OK LOGOUT completed
>> Command 5 took 0.0759 seconds.
------------------------------
>> Sat, 28 Apr 2018 04:48:14 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.2304 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT ID CHILDREN NAMESPACE IDLE UIDPLUS AUTH=PLAIN] Fenix ready.
C: 1 [INITIAL CLIENT RESPONSE (username: comms@REMOVED.org)]
S: +
C: Y29tbXNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW1zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] LOGIN Full IMAP support is enabled
>> Command 1 took 0.2528 seconds.
C: 2 SELECT INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 2] Predicted next UID
S: 2 OK [READ-WRITE] SELECT completed.
>> Command 2 took 0.0804 seconds.
C: 3 UID STORE 1 +FLAGS \seen
S: 3 NO Error in IMAP command received by server.
>> Command 3 took 0.076 seconds.
C: 4 LOGOUT
S: * BYE IMAP server shutting down
S: 4 OK LOGOUT completed
>> Command 4 took 0.0759 seconds.

```

</details>

##### SMTP log (data/horde_smtp.log)

Sending mail is fine. It's IMAP that is fubar. 

##### Browser log
<details>

```

JQMIGRATE: Migrate is installed, version 1.4.0
core.js:7:542

Starting Mail …
build.js:91:8433

Source map error: SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
Resource URL: https://board.REMOVED.org/core/vendor/core.js?v=688d2d1f-0
Source Map URL: purify.min.js.map[Learn More]
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: call to eval() or related function blocked by CSP.
6aa727db-9c0f-4c48-8e43-ecfc4a61ea19:27

```

</details>

**Comment by ChristophWurst** [comment]
Please trim your IMAP logs to the relevant communication where the message is deleted. I couldn't find that section right now.

Is this just a temporary issue or does the messages still show up after a full page refresh? That should give us an indication if it's a IMAP-related bug or just some faulty state handling on the client-side.

**Comment by transmutated** [comment]
This is ongoing. If I refresh the page after I delete it, it shows back up again. Here is the appropriate log:

```
------------------------------
>> Mon, 30 Apr 2018 20:06:06 +0000
>> Connection to: imap://imap.REMOVED.net:993/
>> Server connection took 0.284 seconds.
S: * OK [CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN] Courier-IMAP ready. Copyright 1998-2004 Double Precision, Inc.  See COPYING for distribution information.
C: 1 [INITIAL CLIENT RESPONSE (username: communications@fairfieldfarms.org)]
S: +
C: Y29tbXVuaWNhdGlvbnNAZmFpcmZpZWxkZmFybXMub3JnAGNvbW11bmljYXRpb25zQGZhaXJmaWVsZGZhcm1zLm9yZwBGdWNrVGhlUG9saWNlMjAxOCE=
S: 1 OK LOGIN Full IMAP support is enabled
>> Command 1 took 0.18 seconds.
C: 2 CAPABILITY
S: * CAPABILITY IMAP4rev1 UNSELECT STARTTLS ID CHILDREN NAMESPACE IDLE AUTH=PLAIN
S: 2 OK CAPABILITY completed
>> Command 2 took 0.2031 seconds.
C: 3 LIST "" *
S: * LIST (\NoInferiors) "/" "INBOX"
S: * LIST (\HasNoChildren) "/" "Archives"
S: * LIST (\NoInferiors \Junk) "/" "Bulk Mail"
S: * LIST (\NoInferiors) "/" "Drafts"
S: * LIST (\HasNoChildren) "/" "Email_Templates"
S: * LIST (\NoInferiors) "/" "Send_Later"
S: * LIST (\NoInferiors \Sent) "/" "Sent Items"
S: * LIST (\HasNoChildren \Trash) "/" "Trash"
S: 3 OK  Success
>> Command 3 took 1.689 seconds.
C: 4 SELECT INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 1297] Predicted next UID
S: 4 OK [READ-WRITE] SELECT completed.
>> Command 4 took 4.4125 seconds.
C: 5 UID COPY 1289 Trash
S: 5 OK [COPYUID 1 1289 107] Done
>> Command 5 took 1.6747 seconds.
C: 6 UID SEARCH NOT UID 1289 DELETED
S: * SEARCH
S: 6 OK SEARCH done.
>> Command 6 took 0.5203 seconds.
C: 7 UID STORE 1289 +FLAGS \deleted
S: 7 NO Error in IMAP command received by server.
>> Command 7 took 0.0846 seconds.
C: 8 CLOSE
S: 8 OK mailbox closed.
>> Command 8 took 0.0822 seconds.
C: 9 SELECT INBOX
S: * FLAGS (\Draft \Answered \Flagged \Deleted \Seen)
S: * OK [PERMANENTFLAGS (\Draft \Answered \Flagged \Deleted \Seen)] Limited
S: * 1 EXISTS
S: * 0 RECENT
S: * OK [UIDVALIDITY 1] Ok
S: * OK [UIDNEXT 1297] Predicted next UID
S: 9 OK [READ-WRITE] SELECT completed.
>> Command 9 took 0.4383 seconds.
C: 10 LOGOUT
S: * BYE IMAP server shutting down
S: 10 OK LOGOUT completed
>> Command 10 took 0.0818 seconds.
```

**Comment by ChristophWurst** [comment]
> C: 7 UID STORE 1289 +FLAGS \deleted
> S: 7 NO Error in IMAP command received by server. 
> Command 7 took 0.0846 seconds.

Thanks for the log. This is where we've got an IMAP error.

**Comment by enricogalli** [comment]
I had a similar problem, email messages coul not be deleted and an error message telling "unable to delete" was displayed on top of the screen. 
I noticed that my mail service didn't have a /Trash IMAP folder.
I added a /Trash IMAP Folder and now everything works fine.

This is more a workaround than a solution but I hope it helps.

**Comment by ChristophWurst** [comment]
I'm closing this ticket because it hasn't received any updates in a long time. If this is still relevant, feel free to re-open or file a new ticket.

Thank you.
